vspace/src/main/java/edu/asu/diging/vspace/web/staff/forms/ExhibitionConfigurationForm.java

```diff
@@ -0,0 +1,64 @@
+package edu.asu.diging.vspace.web.staff.forms;
+
+import java.util.List;
+
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+
+import edu.asu.diging.vspace.core.model.ExhibitionModes;
+
+public class ExhibitionConfigurationForm {
+    
+    private String customMessage;
+    
+    private String defaultExhibitLanguage;
+    
+    @NotEmpty(message = "Please select exhibition languages")
+    private List<String> exhibitLanguage;
+    
+    @NotNull(message = "Please select exhibit mode")
+    private ExhibitionModes exhibitMode;
```

Comment:
there is not really a reason to save 3 characters, just write out "exhibition" (and in the message it definitely need to be written out)